src/components/PostContainer/PostContainer.js

```diff
@@ -0,0 +1,28 @@
+import React, {Component} from 'react';
+import CommentSection from '../CommentSection/CommentSection.js';
+import dummyData from '../dummy-data.js';
+
+const PostContainer = (props) => {
+	return(
+		<div className="Container">
+			<div className="Wrapper">
+				<img src={dummyData.thumbnailUrl}/>
+				{dummyData.username}
+			</div>
+
+			<img className="Post-Img" src={dummyData.imageUrl}/>
```

Comment:
Because you passed down props properly, you can actually do something like `src={props.post.imageUrl}`